scrapers/base.py

```diff
@@ -52,6 +50,8 @@ def scrape(self, url, base_url):
             for item in items:
                 img_src = item.find("img").get("src")
                 img_url = base_url + img_src
+                if not img_url:
+                    img_url = "https://github.com/CodeForAfrica/TaxClock/blob/kenya/img/placeholder.png"
```

Comment:
Does this load as expected? Two things:

- I'd figure we need to work on the raw image URL from Github or better still, a relative URL for the sake of replicability e.g if someone wants to deploy their own TaxClock with their own placeholder image.
- `img_url` won't ever be empty because `base_url` wouldn't either.